app/views/company_branches/new.html.erb

```diff
@@ -1,16 +1,13 @@
-<h1>New branch</h1>
-
-<%= form_with model: CompanyBranch, url: company_branches_path do |form| %>
-  <div class="mb-3">
-    <%= form.label :name%>
-    <%= form.text_field :name, class: "form-control"%>
-  </div>
-  <div class="mb-3">
-    <%= form.label :address%>
-    <%= form.text_field :address, class: "form-control"%>
-  </div>
-  <div class="mb-3">
-    <%= form.submit "Add company", class: "btn btn-primary" %>
-    <%= link_to "Back", company_branches_path , method: :get, class: "btn btn-primary" %>
-  </div>
-<% end %>
+<div class="col-5">
```

Comment:
You should wrap this inside a container and a row.
See:  [bootstrap docs](https://getbootstrap.com/docs/5.0/layout/grid/).

Reply:
Of course!

Reply:
Of course , but this is already done in the layout:
https://github.com/BC-MAR-21-ROR/kata-07-rejoj-checador-equipo-03-k07/blob/master/app/views/layouts/application.html.erb#L15-L16 .
Doing this in the layout we can avoid repeating in every view.